src/client/dfs.cc

```diff
@@ -976,6 +978,7 @@ uint64_t DFS::read(std::string& file_name, char* buf, uint64_t off, uint64_t len
 
     remain_len -= io_ops.length;
 
+    // What is it??
     if(io_ops.pos + io_ops.length > fd->block_size[i])
```

Comment:
Its seems that you were the last one to write that line, maybe some security check?